README.md

```diff
@@ -1,10 +1,28 @@
 # GitHub Gems: Driving Open-Source Investments With Data
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+## Project Title and Description
```

Comment:
Minor nit-pick: Maybe rename this to "Overview" and remove the mention of "project" and "driving open-source investments" to make it look less academic?